python/example_code/stepfunctions/sfn_create_state_machine.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import boto3
+
+#The name of the state machine
+SF_NAME = 'name'
+
+# The Amazon States Language definition of the state machine
+SF_DEFINITION = 'definition'
```

Comment:
There seems to be an error with the definition.

Unrecognized token 'definition': was expecting ('true', 'false' or 'null')

Please review.

If there are only specific acceptable inputs, then list as 'value1' | 'value2'

Thank you.

Reply:
Thanks jschwarzwalder for the review , i will check these changes and update